features/checkout/paying_for_order/viewing_available_payment_methods_based_on_current_channel.feature

```diff
@@ -0,0 +1,62 @@
+@paying_for_order
+Feature: Viewing available payment methods based on current channel
+    In order to be able to pay for order in preferable payment method
+    As a Customer
+    I want to see all available for me payment methods
+
+    Background:
+        Given the store operates on another channel named "United States" in "USD" currency and with hostname "usa.cool-clothes.example"
+        And the store operates on a channel named "Poland" with hostname "polski-sklep.pl"
+        And the store has a zone "World"
+        And the store ships to "United States"
+        And this zone has the "United States" country member
+        And the store allows paying "Bank of America" in "United States" channel
+        And the store allows paying "Bank of Poland" in "Poland" channel
+        And the store allows paying offline for all channels
+        And the store allows paying "Bank of Universe"
+        And this payment method has been disabled
+        And the store has a product "PHP T-Shirt" priced at "$19.99" in "United States" channel
+        And this product is also priced at "$25.00" in "Poland" channel
+        And the store ships everywhere for free for all channels
+
+    @ui @api
+    Scenario: Seeing payment methods that are available in channel as an logged in customer
+        Given I am a logged in customer
+        And I view shop on "United States" channel
```

Comment:
This step looks like `Then` but it is used as `Given`, could you change it a little? For example to sth like:
```suggestion
        And I am in the "United States" channel
```